### ISSUE TITLE
Download from southparkstudios.com.br (Brazil) not working

### ISSUE THREAD

**Issue by duduws** [xypwn/southpark-downloader-ui#9]
https://www.southparkstudios.com.br/episodios/940f8z/south-park-cartman-ganha-uma-sonda-anal-temporada-1-ep-1

Can we download frm southparkstudios.com.br ? (Brazilian Portuguese edition)

**Comment by xypwn** [comment]
No, this app doesn't support Brazilian Portuguese at the moment.

If you like, I can tag this as a feature request and implement support for the language when I have time.

**Comment by xypwn** [comment]
If you want to download English episodes in the meantime, you can always use a VPN to do that.
